CharlieBackend.Data/ApplicationContext.cs

```diff
@@ -413,6 +415,21 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                     .HasConstraintName("FK_student_of_visit");
             });
 
+            modelBuilder.Entity<Schedule>(entity =>
+            {
+                entity.HasIndex(e => e.StudentGroupId)
```

Comment:
Maybe its necessary to insert                 
entity.ToTable("schedule");

Because in other way it will create table with name schedules, what does not correspont to our DB syntax

Reply:
done